File: Assets/Loads/Loads.cs
using UnityEngine;

namespace Grabby
{
    [CreateAssetMenu(fileName = "New Loads", menuName = "Grabby/Loads")]
    public class Loads : ScriptableObject
    {
        public Load[] items;
    }

    public struct LoadsCollection
    {
        public int turtle;
        public int shark;
        public int teddy;
        public int octopus;
    }
}

File: Assets/Server/Common/loads.js
const loads = [
    {
        name: 'octopus'
    },
    {
        name: 'shark'
    },
    {
        name: 'teddy'
    },
    {
        name: 'turtle'
    }
];
export const loadNames = loads.map(load => load.name);

export function getLoadsCollectionSchema(value) {
    const schema = {};
    loads.forEach(load => schema[load.name] = value);
    return schema;
}

export default loads;